pitest-maven/src/main/java/org/pitest/maven/SurefireConfigConverter.java

```diff
@@ -79,4 +80,11 @@ private void convertExcludes(ReportOptions option, Xpp3Dom configuration) {
     return Collections.emptyList();
   }
 
+  private void convertTestFailureIgnore(ReportOptions option, Xpp3Dom configuration) {
+    List<String> testFailureIgnoreList = extractStrings("testFailureIgnore", configuration);
+    if (!testFailureIgnoreList.isEmpty()) {
```

Comment:
what happens if the list contains more than 1 element? is it ignored? should we log?

Reply:
I could just skip the list conversion and directly extract the value.